src/routes/(app)/product/[id]/+page.server.ts

```diff
@@ -118,7 +137,8 @@ export const load = async ({ params, locals }) => {
 			productCMSAfter: cmsFromContent({ content: product.contentAfter }, locals)
 		}),
 		showCheckoutButton: runtimeConfig.checkoutButtonOnProductPage,
-		websiteShortDescription: product.shortDescription
+		websiteShortDescription: product.shortDescription,
+		jsonLd
```

Comment:
Instead of passing it to the frontend like this, is there a reason for not defining it directly in the frontend?

Reply:
There is not. I just put it to make code readable

Reply:
If it's readability we could maybe create the function to convert product to JSON-LD in `src/lib/types/Product.ts` 

But let's avoid doing the frontend's work on the backend.

Up to you wheter to put it directly in `+page.svelte` or in `src/lib/types/Product.ts`